python/adelphi/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27,py36
```

Comment:
It would be great if we could specify only the major versions here like `py2,p3`. Instead of py36, I only have py39 locally, so it failed the first time I tried to run the tests and I had to change this setting manually.

Reply:
This is a tox thing, and I'm not sure how much control we have over it.  According to their docs (https://tox.readthedocs.io/en/latest/config.html) "py2" and "py3" should be supported.  Did you try those explicitly and see them fail?

I'll also give them a run locally and see if I can make them work.

One thing I did notice about tox; if you specify a Python version higher than the one you currently support you get errors about an inability to create a venv.  I have 3.6.x on my local box (that's where the "py36" came from) and when I tried to do a test with "py39" is complained about an inability to grab the interpreter.  So it's not like it's downloading Python versions or anything... this isn't rvm.  But when I make Python 3.9.x my active "py3" then "tox -e py36" seems to work okay.  It's... kinda strange; I definitely don't completely understand how tox is working yet.

I'll dig into this a bit more; I agree that if we can get away with "py2" and "py3" here that's better.  Well, we probably should require "py27" since we aren't supporting anything else in the 2.x line... but you see what I'm driving at.

Reply:
I just tried "py3" and it works 👍 
But "py2" doesn't, it fails with: `ERROR: InterpreterNotFound: python2`, but as you said, it's probably ok to have it as py27.

Reply:
This is still an issue for me. From a clean checkout, it fails because I don't have py36. I have to manually set it to py3.
Does it work with py3 for you? If so, any objections with going with that by default?

Reply:
Yeah, "py3" should work just fine... and should generalize better.  I'll remove the explicit minor version designation.